android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt

```diff
@@ -146,7 +146,7 @@ class ModelViewer(
 
     constructor(
             surfaceView: SurfaceView,
-            engine: Engine = Engine.create(),
+            engine: Engine = Engine.Builder().build(),
```

Comment:
Don't change this.

Reply:
But there are warnings about `deprecated` api.
So, should the `@Deprecated` annotation be removed from `Engine.create`?

Reply:
Apparently that `@Deprecated` was recently added, that's why I didn't see it in my source tree. It seems heavy-handed. @poweifeng are we absolutely sure we need to deprecate those APIs? Now all Java apps/samples will have a deprecation warning at compile time for no good reason.

Reply:
I suppose it's ok to *not deprecate*.  I'm kind of following the reasoning in this comment: https://github.com/google/filament/blob/main/filament/include/filament/Engine.h#L371

Though the comment itself does not indicate a deprecation.

Reply:
Okay, I deleted the `@Deprecated` annotation from `Engine.create`.
But I prefer to keep `@Deprecated`, it's the `Java`'s way in my mind.

It will revert part of #7223 .

Reply:
<img width="880" alt="Screen Shot 2023-10-06 at 10 50 12 AM" src="https://github.com/google/filament/assets/2757043/28069c19-6fc8-40f3-b29d-d8ad6896f2f9">

I forgot to revert the `NEW_RELEASE_NOTES.md`, part of #7223  , @poweifeng Do you want me to do it in another PR? or do you want to do it by yourself?

`engine: In Java, decpreate Engine.create() and recommend Engine.Builder instead` ->
`engine: In Java, add Engine.Builder to create Engine instance.`

Reply:
I can make a change. Thanks!